src/main/java/pl/javastart/task/BubbleSort.java

```diff
@@ -4,6 +4,31 @@ public class BubbleSort {
 
     // ta metoda powinna sortować inputArray oraz zwracać informację o liczbie porównań, które były do tego potrzebne
     public int sort(int[] inputArray) {
-        return 0;
+
+        int counter = 0;
+        int tempContainer;
```

Comment:
Nie deklaruj zmiennych "na zapas". Jeżeli ta zmienna jest Ci potrzebna dopiero wewnątrz ifa, to tam ją zadeklaruj